third_party/subscriptions-project/config.js

```diff
@@ -599,10 +599,9 @@ class MetaParser {
     // Is locked?
     const accessibleForFree = getMetaTag(this.doc_.getRootNode(),
         'subscriptions-accessible-for-free');
-    let locked = false;
-    if (accessibleForFree && accessibleForFree.toLowerCase() == 'false') {
-      locked = true;
-    }
+    const locked = (accessibleForFree &&
+        accessibleForFree.toLowerCase() == 'false') || false;
```

Comment:
IIRC, the newer version of CC could not determine that this expression result was boolean. Recommend reverting and making this change upstream if it complains.

Reply:
Yep. Now it's fine.